### DISCUSSION TITLE
Pregenerated chunks?

### DISCUSSION THREAD

**Discussion by vrogojin** [mircokroon/minecraft-world-downloader#146] [Q&A]
Hi!
Do I understand correctly that if I set level seed and render distance greater than the server's one, chunks that are not received from server are being pregenerated by the MWD by default MC terrain generation algorithm?
Well, it does not seem to pregenerate chunks for me, or am I doing something wrong?

Thanks!

**Comment by mircokroon** [comment]
Not quite, it will only send previously downloaded chunks to the client. Generating chunks would require the entire chunk generation code to be included I think which isn't feasible.
